.idea/vcs.xml

```diff
@@ -7,6 +7,10 @@
           <option name="issueRegexp" value="[A-Z]+\-\d+" />
           <option name="linkRegexp" value="https://issues.apache.org/jira/browse/$0" />
         </IssueNavigationLink>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="(apache/calcite)?#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/apache/calcite/pull/$2" />
+        </IssueNavigationLink>
```

Comment:
I think most commits don't have such a message. But I think it is still useful.

Reply:
Some of them have. :) 🍡 
![image](https://user-images.githubusercontent.com/15352793/78852230-63f7ad00-79e1-11ea-97e3-e3be85f37327.png)